src/Entity/User.php

```diff
@@ -17,10 +18,16 @@
 class User extends AbstractBaseEntity implements UserInterface, PasswordAuthenticatedUserInterface, \JsonSerializable, TenantScopedUserInterface
 {
     /**
-     * @ORM\Column(type="string", length=180, unique=true)
+     * @ORM\Column(type="string", unique=true)
+     */
+    #[Assert\NotBlank]
+    private string $providerId = '';
+
+    /**
+     * @ORM\Column(type="string", length=180, nullable=true)
```

Comment:
Even with a `$providerId` I still don't think `$email` should be nullable

Reply:
And we should still enforce `unique=true` on emails

Reply:
Fixed